### PR TITLE
mar dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-march-2022.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+The legacy PAM authentication plugin has been ported to the new Authentication Plugin Framework. Although the legacy PAM auth plugin hasn't been merged, the fact that we were able to port it helps build confidence in the auth plugin framework. Our next task is to allow for more dynamic PAM conversations.
+
+The metadata API endpoint has received a new enhancement in that it now supports the admin mode flag. This change is important because it simplifies implementations (e.g. Logical Quotas). There are many times where code needs to manipulate metadata, but doesn't have permission to do so. Getting around this case requires modifying permissions temporarily. With this change, that isn't required anymore. And obviously, we made sure to expose this option to administrators via `imeta -M` :-).
+
+Since we're still talking about the server, we're happy to say that the Python Rule Engine Plugin now supports binary I/O.
```
Comment (trel) [inline content/posts/irods-development-update-march-2022.md:20]: suggestion...

"binary I/O" -> "reading and writing binary data into data objects."

File: content/posts/irods-development-update-march-2022.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+
+Let's talk about what's happened on the client-side now ...
+
+The Python iRODS Client (PRC) has grown several new features. It now supports operations found in `ipasswd`, provides utilities for path normalization, and here is the big one ... it supports parallel transfer to and from S3 storage! Even more important than that is the fact that once this version is released, it will likely be the last to support Python 2.
+
+New versions of the Zone Management Tool (ZMT) and C++ REST API were released this month. With them comes support for managing remote zones, remote users, checkfiles (default and custom), specific queries, and passwords.
+
+Outside of servers and clients, the development and testing environments have gained support for new platforms! Support for Ubuntu 20, Debian 11, and AlmaLinux 8 have been added. If we're lucky, that means we get support for Rocky Linux 8 for free. The development environment also grew the ability to build externals packages. Hopefully, that simplifies the build process for users.
```
Comment (trel) [inline content/posts/irods-development-update-march-2022.md:28]: "if we're lucky" .... no, this is the design :)

File: content/posts/irods-development-update-march-2022.md [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+- (dan) made it so that docker images are automatically rebuilt when necessary.
+- (dan) added support for binary I/O to the PREP.
+- (kory) released new C++ REST version 0.9.1 with support for changing passwords.
+    - added support for group permissions to atomic metadata / ACLs APIs
+    - added support for the admin flag to imeta and the metadata C APIs
+    - enabled support for C++20 (main branch only)
+- (bo) added support to ZMT for remote zones and users.
+- (bo) added support to ZMT for default and custom checkfiles.
+- (bo) added support to ZMT for managing specific queries.
+//// THIS WILL BE REMOVED ////
```
Comment (trel) [inline content/posts/irods-development-update-march-2022.md:53]: remove

**Review by trel — COMMENTED** [review]
looking good.

File: content/posts/irods-development-update-march-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-march-2022.md:20]: will change

File: content/posts/irods-development-update-march-2022.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-march-2022.md:28]: will change

File: content/posts/irods-development-update-march-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-march-2022.md]: done.

File: content/posts/irods-development-update-march-2022.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-march-2022.md]: done.

File: content/posts/irods-development-update-march-2022.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-march-2022.md]: done.

**Comment by korydraughn** [comment]
I've changed the about Rocky Linux 8. I think we're good to go.

**Comment by trel** [comment]
i think so.   will merge it tomorrow AM.

**Pull request merged**